modular-docs-manual/content/topics/module_creating-assemblies.adoc

```diff
@@ -0,0 +1,32 @@
+[[creating-assemblies]]
+= Creating Assemblies
+
+An assembly is a representation of a user story, so you need to create an assembly for each user story that you defined.
+
+An assembly could represent an article, "chapter" in a book, or even an entire book. However, one of the benefits of modular documentation is that you do not need to worry about how the assembly will ultimately be used -- each assembly represents a user goal, and once you create it, it can be "included" anywhere it is needed (a publication, within another assembly, and so on).
+
+.Procedure
+
+. Create an assembly file for each user story that you identified.
++
+--
+Be sure to follow a consistent file naming standard. For example, you could use the following file naming pattern:
+
+`assembly_configuring-security.adoc`
+--
+
+. For each top-level assembly file that you created, fill in the content:
++
+--
+* Title
+* Intro paragraphs
+* Prerequisites
+* Modules or Assemblies
++
+If the top-level assembly has second-level assemblies, then add the names of the second-level assemblies. Otherwise, list the modules.
+* Additional resources/Related information
+
+For more information about forming assemblies, see xref:assembly-guidelines[Assembly Guidelines].
+--
+
+. Repeat the previous step for each second-level assembly.
```

Comment:
One question popped into my head: What about the top-level assemblies? If I'm being asked to create assembly files for second-level assemblies only, I'm not sure what to do with the top-level ones. Do I discard them? Or will they somehow appear in the table of contents?

Reply:
How about another example box that would show the source of an assembly?

~~~~
[#creating-network-connections]
= Creating Network Connections

You can create network connections to enable Product X to accept and make connections to other endpoints.

include::concept-types-of-connections[leveloffset=+1]
include::procedure-creating-listeners-to-accept-incoming-connections[leveloffset=+1]
include::procedure-creating-connectors-to-connect-to-outbound-endpoints[leveloffset=+1]
include::reference-network-connection-configuration-attributes[leveloffset=+1]

~~~~

Reply:
As previously stated, this would be good info to add if we want to get AsciiDoctor-specific.